### PR TITLE
v0.9.47

### PR DESCRIPTION
**Pull request by dmgav**
Removed `tomopy` from requirements

Updated some dependencies

### CODE REVIEW

**Comment by mrakitin** [comment]
The builds are failing due to one of the patches are already in the code, and not needed (see the full building log [here](https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=2403&view=logs&j=4b3757f9-99c5-5891-6c0c-93d7030cdf7d&t=0a2070c1-8027-59fc-ddea-6aec050600c1&l=83)):
```
INFO conda_build.source:apply_patch(598): Trying to apply patch as-is
patching file larch/apps.py
Hunk #1 FAILED at 2.
Hunk #2 FAILED at 68.
Hunk #3 FAILED at 82.
Hunk #4 succeeded at 150 (offset 17 lines).
Hunk #5 FAILED at 191.
4 out of 5 hunks FAILED -- saving rejects to file larch/apps.py.rej
```

**Comment by dmgav** [comment]
I think I fixed the issues with patches, but now it's in the same state as https://github.com/nsls-ii-forge/xraylarch-feedstock/pull/3: the cases for OSX and Windows with Python 3.6 do not pass. It also looks like Python 3.7 is getting installed in the environments instead of Python 3.6.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,29 +30,30 @@ requirements:
...
     - silx
     - six >=1.10
     - sqlalchemy >=0.9
     - termcolor
-    - tomopy
     - uncertainties
     - wxmplot
     - wxpython
     - wxutils
+    - xraydb >=4.3
```
Comment (mrakitin) [inline recipe/meta.yaml:56]: @dmgav, where did these pins/new packages come from? I'll probably need to port them to #4.

**Comment by mrakitin** [comment]
Superseded by #4.

**Pull request closed without merging**